main.go

```diff
@@ -0,0 +1,52 @@
+package main
+
+import (
+	"bufio"
+	"fmt"
+	"log"
+	"os"
+	"strings"
+	"strconv"
+)
+
+func main() {
+	file, err := os.Open("instance.sat")
+	if err != nil {
+		log.Fatal(err)
```

Comment:
I've heard people say 'log.Fatal' should be deprecated, but I don't know what the correct thing to do is here, so let's keep it unless you find something.